### PR TITLE
fix:admin tab loading

### LINKED ISSUE #810: [Bug]: Admin tabs do not load data until you refresh
### What happened?

Often times when you load admin portal the tabs are either stuck in a loading state or nothing appears until you refresh.

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by shlok-2003**
## Description
Removed some return statments

## Related Issue
Closes #810

### CODE REVIEW

**Comment by bautistaaa** [comment]
i dont understand how this fixes the problem

**Comment by shlok-2003** [comment]
Hey @bautistaaa, after these changes, the following issue is not being reproduced on my localhost. Is there any way to test it more further

File: apps/admin/src/app/_components/tracks/index.tsx [hunk 1]
```diff
@@ -6,14 +6,20 @@ import { ActionBar } from './action-bar';
...
+
 export function ManageTracks() {
   const { data, isLoading } = useQuery(['admin-tracks'], () => getTracks());
-  if (isLoading || !data) return <div>Loading...</div>;
 
   return (
     <div>
-      <ActionBar />
-      <DataTable data={data} columns={columns} />
+      {!data || isLoading ? (
```
Comment (ja153903) [inline apps/admin/src/app/_components/tracks/index.tsx:15]: This effectively doesn't change anything. It just moves the condition from its own if-statement down to the return

File: apps/admin/src/app/_components/challenge-reviews/index.tsx [hunk 2]
```diff
@@ -8,11 +8,9 @@ import { columns } from './columns';
 export function ChallengeReviews() {
   const { data, isLoading } = useQuery(['challenge-reviews'], () => getPendingChallenges());
 
-  if (isLoading || !data) return null;
-
   return (
     <div className="w-full rounded-md border">
-      <DataTable data={data} columns={columns} />
+      {data ? <DataTable data={data} columns={columns} /> : ''}
```
Comment (ja153903) [inline apps/admin/src/app/_components/challenge-reviews/index.tsx:13]: Similar to the other change, you're just displacing where the empty check is. This doesn't solve the supposed issue.

**Review by ja153903 — COMMENTED** [review]
@bautistaaa I think we need a reliable way to reproduce this bug.

**Comment by shlok-2003** [comment]
Ok, I'll see if I can find any errors in the api's

**Comment by bautistaaa** [comment]
> @bautistaaa I think we need a reliable way to reproduce this bug.

if there was i would have written it. unfortunately its mainly on initial load. its not high priority as its admin only just annoying

**Comment by shlok-2003** [comment]
Hey @ja153903, as of now, I am unable to find any issues in the codebase.

**Pull request closed without merging**